lib/app/views/application/_nav.erb

```diff
@@ -2,6 +2,22 @@
   <ul class="nav navbar-nav">
     <li><a href="http://help.exercism.io">Help</a></li>
     <li><a href="/getting-started">Welcome new users</a></li>
+    <li class="dropdown">
+      <a class="dropdown-toggle" id="drop-current" href="#" data-toggle="dropdown">Languages<b class="caret"></b></a>
+    <ul class="dropdown-menu" id="menu-current" role="menu" area-labelledby="drop-current">
+          <% all_languages.each do |l| %>
```

Comment:
We discussed that and had pretty much the same reaction: we like the second (tribe/monkey) and third (monkeys/monkey) options better, with a slight preference for the third one. They are both self-explanatory, but `tribe` might need some additional background to make sense. `monkeys` is clearly a group of single `monkey`, but `tribe` might be a group with specific attributes -- is one tribe of monkeys equivalent to another tribe? Do they have a specific behavior? Do only monkeys with tails belong to tribes?